pkg/render/amazoncloudintegration.go

```diff
@@ -315,8 +315,8 @@ func (c *amazonCloudIntegrationComponent) container() corev1.Container {
 				},
 			},
 			InitialDelaySeconds: 10,
-			PeriodSeconds:       10,
-			FailureThreshold:    3,
```

Comment:
Fields set with default values are removed. `FailureThreshold: 3` is the default value.

Reply:
I think we should explicitly set these; k8s changed their defaults (from a 10+s timeout to 1s) before and that's how we ended up with liveness probes failing.

Reply:
It is explicitly set now.